src/main/java/org/jenkins/tools/test/hook/AbstractMultiParentHook.java

```diff
@@ -45,19 +43,13 @@ public Map<String, Object> action(Map<String, Object> moreInfo) {
                         new File(config.workDirectory.getAbsolutePath() + "/" + getParentFolder());
 
                 pomData = (PomData) moreInfo.get("pomData");
-                String scmTag;
-                if (pomData.getScmTag() != null) {
-                    scmTag = pomData.getScmTag();
-                    LOGGER.log(Level.INFO, "Using SCM tag {0} from POM", scmTag);
-                } else {
-                    scmTag = getParentProjectName() + "-" + currentPlugin.version;
-                    LOGGER.log(
-                            Level.INFO,
-                            "POM did not provide an SCM tag; inferring tag {0}",
-                            scmTag);
-                }
-                // Like PluginCompatTester.cloneFromSCM but with subdirectories trimmed:
-                cloneFromSCM(parentPath, scmTag, getUrl(), config.getFallbackGitHubOrganization());
+                // Like the call in PluginCompatTester#runHooks but with subdirectories trimmed:
+                PluginCompatTester.cloneFromScm(
+                        getUrl(),
+                        config.getFallbackGitHubOrganization(),
+                        PluginCompatTester.getScmTag(
+                                pomData, getParentProjectName(), currentPlugin.version),
+                        parentPath);
```

Comment:
Minor cleanup while I was here: deduplicating this code with that in `PluginCompatTester`.